.github/workflows/ci.yml

```diff
@@ -156,5 +156,6 @@ jobs:
           nim --version
           nimble --version
           nimble install -y --depsOnly
+          rm -f nimble.lock
```

Comment:
just remove the file instead, it causes the same problems locally

Reply:
okay